NebulaNetwork/PacketProcessors/Factory/Entity/BuildEntityRequestProcessor.cs

```diff
@@ -13,9 +12,9 @@ public class BuildEntityRequestProcessor : PacketProcessor<BuildEntityRequest>
     {
         public override void ProcessPacket(BuildEntityRequest packet, NebulaConnection conn)
         {
-            if (IsHost && !FactoryManager.ContainsPrebuildRequest(packet.PlanetId, packet.PrebuildId))
+            if (IsHost && !FactoryManager.Instance.ContainsPrebuildRequest(packet.PlanetId, packet.PrebuildId))
             {
-                Log.Warn($"BuildEntityRequest received does not have a corresponding PrebuildRequest with the id {packet.PrebuildId} for the planet {packet.PlanetId}");
+                //Log.Warn($"BuildEntityRequest received does not have a corresponding PrebuildRequest with the id {packet.PrebuildId} for the planet {packet.PlanetId}");
```

Comment:
Why is this commented out?

Reply:
I was commenting these lines during testing, i think i forgot about this one.